new/PRS14029Eraqlis.xml

```diff
@@ -47,17 +47,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <person sex="1">
                    
                     <persName xml:lang="gez" xml:id="n1">ኤራቅሊስ</persName>
-                    <persName xml:lang="gez" type="normalized" corresp="#n1"> Ǝrāqis</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"> <idno>MS GG no. 00044</idno>ʿErāqis</persName>
```

Comment:
I think the insertion of the manuscript's ID here was a copy paste error. Here, we only need the correct transliteration of ኤራቅሊስ. Please, delete `<idno>` and correct the transliteration.